pageserver/src/pgdatadir_mapping.rs

```diff
@@ -1392,6 +1392,37 @@ impl<'a> DatadirModification<'a> {
         content: &[u8],
         ctx: &RequestContext,
     ) -> anyhow::Result<()> {
+        const AUX_FILES_V2: bool = false; // disable for now until we settle down the feature flag for aux file v2
```

Comment:
Hm. I don't feel like we should merge dead code. IMO there should be at least a unit test exercising it.